src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector128_1.cs

```diff
@@ -74,6 +75,7 @@ internal static bool IsSupported
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Need this on both the throw and the IsSupported check or the JIT will still emit a call.